mix_install/bench_insert.exs

```diff
@@ -0,0 +1,43 @@
+Mix.install([
+  {:ch, github: "plausible/ch", branch: "hexdocs"},
```

Comment:
```suggestion
  {:ch, path: "."},
```

Reply:
I'd like these to be executable outside of this git repo: https://github.com/plausible/ch/tree/hexdocs#benchmarks

Or is it too much? `curl` vs `git clone`...

Reply:
feels like too much, this way the benchmarks are runnable against your current local copy so you can tweak code as you re-run them?

Reply:
yeah, right now I actually have this part commented out and use MIX_ENV=bench mix run mix_install/bench_insert.exs`